### ISSUE TITLE
Update Partner Page

### ISSUE THREAD

**Issue by PeterSuma** [abr/abr.github.io#11]
Add DARPA, spinnaker, westwell to the partner page.

**Comment by krisfrenette** [comment]
I don't think that we currently have a partner page on the new website. Is this something that we would like to add?
